src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -297,10 +299,17 @@ export class OsmosisSwapper implements Swapper<ChainId> {
         feeDenom: 'uatom',
       })
 
+      const initiatingChainAccountId = toAccountId({ chainId: cosmosChainId, account: sellAddress })
+      const initiatingChainTxid = serializeTxIndex(initiatingChainAccountId, tradeId, sellAddress)
+
       cosmosIbcTradeId = tradeId
 
       // wait till confirmed
-      const pollResult = await pollForComplete(tradeId, cosmosUrl)
+      const pollResult = await pollForCrossChainComplete({
+        initiatingChainTxid,
+        initiatingChainAccountId,
+        getState: store.getState,
+      })
```

Comment:
Implementation was originally brought to single-hop swapper, kept it here so that both use the correct implementation, but this is effectively dead code now